### DISCUSSION TITLE
Current billing cycle row reads and writes?

### DISCUSSION THREAD

**Discussion by Razzmatazzz** [planetscale/discussion#583] [General]
On the database overview page, it used to display the row reads and writes for the current billing cycle. That information appears to have been removed from that page, and I can't find where it's been moved to. Does anyone know where I can find it now?

**Comment by mscoutermarsh** [comment]
Hi @Razzmatazzz - take a look here: https://app.planetscale.com/~/settings/billing/latest

**Reply by Razzmatazzz** [reply]
Thanks. That isn't ideal because only the account administrator can monitor usage.

**Reply by mscoutermarsh** [reply]
Good point, we'll look into it!
